pkgs/by-name/sh/shaderbg/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  cmake,
+  pkg-config,
+  wayland-scanner,
+  wayland,
+  libGL,
+}:
+stdenv.mkDerivation {
+  pname = "shaderbg";
+  version = "0-unstable-2024-11-26";
+  src = fetchFromGitHub {
+    owner = "Mr-Pine";
+    repo = "shaderbg";
+    rev = "cf9b135069550f8d7c8411d8a53285882034331c";
+    hash = "sha256-J+fRzSTEMlT9oLVZqbstrDxuKiJTAHNTLxB8IPgdom0=";
+  };
+  nativeBuildInputs = [
+    meson
+    ninja
+    cmake
+    pkg-config
+    wayland-scanner
+  ];
+  buildInputs = [
+    wayland
+    libGL
+  ];
+  meta = {
+    description = "Live wallpaper program for Sway and other wlr-layer-shell supported compositors that renders shaders as wallpaper";
```

```suggestion
    description = "Shader-based live wallpaper program for compositors that support wlr-layer-shell (Sway and others)";
```
Main selling point should not be at the end in my opinion